chaoswm/actions.py

```diff
@@ -0,0 +1,151 @@
+# -*- coding: utf-8 -*-
+from logzero import logger
+
+from chaoslib.types import Configuration
+from sky_wiremock.sky_wiremock import Wiremock
+from .utils import get_wm_params, check_configuration
+
+__all__ = [
+        "add_mappings",
+        "populate_from_dir",
+        "delete_mappings",
+        "global_fixed_delay",
+        "global_random_delay",
+        "fixed_delay"
+        "random_delay",
+        "chunked_dribble_delay",
+        "down",
+        "up",
+        "reset"
+]
+
+
+def add_mappings(mappings: list=[], configuration: Configuration={}):
```

Comment:
In Python setting function arguments to mutable values is a common error :)
This will have nasty side effects. Better setting the default values to `None` and then check, in the function like this:

`
mappings = mappings or []
`